templates/strings/objc-m.stencil

```diff
@@ -0,0 +1,80 @@
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
+{% if tables.count > 0 %}
+
+#import "Localizable.h"
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wformat-security"
+
+@interface BundleToken : NSObject
+@end
+
+@implementation BundleToken
+@end
+
+NSString* tr(NSString* tableName, NSString* key, ...) {
+    va_list args;
+    va_start(args, key);
+
+    NSBundle *bundle = [NSBundle bundleForClass:BundleToken.class];
+    NSString *format = [bundle localizedStringForKey:key value:@"" table:tableName];
+    NSString *result = [[NSString alloc] initWithFormat:format arguments:args];
+    return result;
+};
+
+{% macro parametersBlock types %}{% filter removeNewlines:"leading" %}
+  {% for type in types %}
+    ({% call paramTranslate type %})p{{ forloop.counter }}{% if not forloop.last %} and:{% endif %}
+  {% endfor %}
+{% endfilter %}{% endmacro %}
+{% macro argumentsBlock types %}{% filter removeNewlines:"leading" %}
+  {% for type in types %}
+    p{{forloop.counter}}{% if not forloop.last %}, {% endif %}
+  {% endfor %}
+{% endfilter %}{% endmacro %}
+{% macro paramTranslate swiftType %}
+  {% if swiftType == "Float" %}
+    float
+  {% elif swiftType == "Int" %}
+    NSInteger
+  {% elif swiftType == "CChar" %}
+    char
+  {% elif swiftType == "String" %}
+    NSString*
+  {% elif swiftType == "UnsafePointer<CChar>" %}
+    char*
+  {% elif swiftType == "UnsafeRawPointer" %}
+    void*
+  {% else %}
+    objc-m.stencil is missing '{{swiftType}}'
+  {% endif %}
+{% endmacro %}
+{% macro recursiveTableContents table item sp %}
+{{sp}}  {% for string in item.strings %}
```

Comment:
You need the `sp` variable, you have a flat template, not a recursive structured one (see `flat-swift4` for example).

Reply:
Do you mean I "**don't** need the `sp` variable"?

Reply:
Uhm, yeah 😅

Reply:
roger!  I'll remove it.